.github/send_alerts_to_slack.py

```diff
@@ -0,0 +1,95 @@
+"""
+
+Summarize the Frictionless Data JSON error output and if there are any errors, 
+send a summary message to a given Slack channel. Send a more detailed 
+error summary to std out and create a detailed txt file that can be accessed via GitHub
+artifacts.
+
+To run elsewhere, define SLACK_WEBHOOK in your repo's secrets and change the 
+top-level constants defined below. 
+
+`validate-csv.yml` can also be copied verbatim, assuming that the schema is 
+named `datapackage.json`.
+
+Links to...
+- Generate a Slack token
+    https://slack.com/apps/A0F7XDUAZ-incoming-webhooks
+- API for POST requests / formatting text:
+    https://api.slack.com/messaging
+
+
+"""
+
+import json
+import requests
+import os
+
+ERROR_FILE = 'error_summary.json'
+REPO = 'Princeton-CDH/pemm-data'
+CHANNEL = '#slack-webhook-test'
+ERROR_MAX = 5
+
+def main():
+    with open(ERROR_FILE) as f: 
+        j = json.load(f)
```

Comment:
Any chance this could fail? Do we need any error handling?

Reply:
please use a more meaningful variable name than `j`

Reply:
fixed `j`, it's now `error_json`, and I assume that python's json error messages are more coherent than anything custom I could write. If the file doesn't exist, python will throw that error. If it's not a valid JSON, python will throw that error. Was there something in particular you were thinking of?